Module2_Class_Logger/Module2_Class_Logger/Logger.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Module2_Class_Logger
+{
+    public class Logger //singleton
+    {
+        private List<Result> _logs = null;
+        private string _path = @"D:\dot_Net_course_A-level\logs.txt";
+        private Logger() { } //private singleton constructor
+
+        private static Logger _instance;
+
+        public static Logger GetInstance()
+        {
+            if (_instance == null)
+            {
+                _instance = new Logger();
+            }
+            return _instance;
+        }
+
+        public Status LogLevel { get; set; }
+
+        public void WriteLog(string consoleOrFile="console")
+        {
+            foreach (Result item in _logs) 
+            {
+                if (consoleOrFile == "console")
```

Comment:
Probably good idea to split it into two private methods: writetoconsole, writetofile